cpp/fetcher/continuous_fetcher.cc

```diff
@@ -107,7 +109,7 @@ void ContinuousFetcherImpl::StartFetch(const unique_lock<mutex>& lock) {
   }
 
   fetch_task_.reset(
-      new Task(bind(&ContinuousFetcherImpl::FetchDone, this, _1), base_));
+      new Task(bind(&ContinuousFetcherImpl::FetchDone, this, _1), executor_));
```

Comment:
The executor for this task is the one that is used to do the disk writes, no need to change anything else.